### ISSUE TITLE
Fatal error on startup

### ISSUE THREAD

**Issue by hlarsson** [docker/for-mac#3816]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: E1A74B40-ECB1-4DA3-B32E-FB12EDBACBB0/20190808080216

### Expected behavior
Should start like normal

### Actual behavior
Startup fails with
Fatal error
The operation couldn’t be completed. (Docker.VmnetdError error 1.)

### Information

  - macOS Version: 10.14.6 
Unfortunately not much information to go on. It started happening after an update of os x to 10.14.6, but the computer had not been restarted in a long time before that, so not sure that is the cause.
I have tried wiping all folders related to docker that i can find and uninstalling and re-installing the app.
Pressing the Reset to factory settings in the error dialog does not help. After restarting, it asks for priviliged access but then the same fatal error appears

### Diagnostic logs

```
Docker for Mac: version...

```

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by hlarsson** [comment]
downgraded to 2.0.0.3 and then it works again

**Comment by pawelkaminski-casumo** [comment]
have the same trouble after upgrading to newest mac os yest.

tl;dr;
```
error	10:17:34.018610 +0200	Docker	SettingsLoader: Cannot load settings: invalidFile
fault	10:17:34.019077 +0200	Docker	Terminator: The operation couldn\U2019t be completed. (Docker.SettingsError error 2.)
default	10:17:34.021477 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
```

full log just after docker starts but when I try hover over docker icon or run any container it hangs and exits
```
default	10:16:38.701523 +0200	com.docker.hyperkit	<--DOCKER-->{"command":"dockerstate","setstate":"running"}
default	10:16:38.701565 +0200	com.docker.hyperkit	
default	10:16:38.817003 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> resuming, QOS(0x19)
default	10:16:38.817269 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> {strength 1, tls 8, ct 0, sub 0, sig 0, ciphers 1, bundle 0, builtin 0}
default	10:16:38.818723 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> now using Connection 1
default	10:16:38.818768 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> sent request, body S
default	10:16:38.995311 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> received response, status 200 content K
default	10:16:38.995467 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> done using Connection 1
default	10:16:38.995554 +0200	Docker	Task <0C8ED33B-8A4A-40A3-AD2A-5725A48936BD>.<11> response ended
default	10:16:38.996161 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> resuming, QOS(0x19)
default	10:16:38.996367 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> {strength 1, tls 8, ct 0, sub 0, sig 0, ciphers 1, bundle 0, builtin 0}
default	10:16:38.996582 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> now using Connection 1
default	10:16:38.996677 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> sent request, body S
default	10:16:39.173471 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> received response, status 200 content K
default	10:16:39.173615 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> done using Connection 1
default	10:16:39.173728 +0200	Docker	Task <2A66FFE1-2608-4B00-858D-466980D0CE94>.<12> response ended
default	10:16:39.174207 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> resuming, QOS(0x19)
default	10:16:39.174422 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> {strength 1, tls 8, ct 0, sub 0, sig 0, ciphers 1, bundle 0, builtin 0}
default	10:16:39.174626 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> now using Connection 1
default	10:16:39.174700 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> sent request, body S
default	10:16:39.351345 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> received response, status 200 content K
default	10:16:39.351496 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> done using Connection 1
default	10:16:39.351611 +0200	Docker	Task <283295E1-3DCE-48C0-B498-3285674E0AD9>.<13> response ended
default	10:17:13.158536 +0200	Docker	TIC TCP Conn Cancel [3:0x600002986640]
default	10:17:13.158607 +0200	Docker	[C3 Hostname#f0a52929:443 tcp, url hash: 51ca64b8, tls] cancel
default	10:17:13.158676 +0200	Docker	[C3 Hostname#f0a52929:443 tcp, url hash: 51ca64b8, tls] cancelled
	[C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443]
	Connected Path: satisfied (Path is satisfied), interface: en0, ipv4, dns
	Duration: 60.778s, DNS @0.000s took 0.356s, TCP @0.359s took 0.124s, TLS took 0.262s
	bytes in/out: 10245/2886, packets in/out: 9/5, rtt: 0.126s, retransmitted packets: 0, out-of-order packets: 0
default	10:17:13.158692 +0200	Docker	0.000s [C3 <private> Hostname#f0a52929:443 resolver] path:start
default	10:17:13.158704 +0200	Docker	0.000s [C3 <private> Hostname#f0a52929:443 resolver] path:satisfied
default	10:17:13.158715 +0200	Docker	0.000s [C3 <private> Hostname#f0a52929:443 resolver] resolver:start_dns
default	10:17:13.158725 +0200	Docker	0.356s [C3 <private> Hostname#f0a52929:443 resolver] resolver:receive_dns
default	10:17:13.158743 +0200	Docker	0.357s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] path:start
default	10:17:13.158755 +0200	Docker	0.357s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] path:satisfied
default	10:17:13.158766 +0200	Docker	0.359s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] flow:start_connect
default	10:17:13.158805 +0200	Docker	0.483s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] flow:finish_transport
default	10:17:13.158848 +0200	Docker	0.483s [C3 <private> Hostname#f0a52929:443 resolver] flow:finish_transport
default	10:17:13.158903 +0200	Docker	0.745s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] flow:finish_connect
default	10:17:13.158921 +0200	Docker	0.746s [C3 <private> Hostname#f0a52929:443 resolver] flow:finish_connect
default	10:17:13.158960 +0200	Docker	0.746s [C3.1 <private> 192.168.66.68:52685<->IPv4#fb370980:443 socket-flow] flow:changed_viability
default	10:17:13.159023 +0200	Docker	0.746s [C3 <private> Hostname#f0a52929:443 resolver] flow:changed_viability
default	10:17:13.159052 +0200	Docker	60.778s [C3] path:cancel
default	10:17:13.159402 +0200	Docker	nw_endpoint_flow_protocol_disconnected [C3.1 IPv4#fb370980:443 cancelled socket-flow (null)] Output protocol disconnected
default	10:17:13.159807 +0200	Docker	nw_connection_report_state_with_handler_locked [C3] reporting state cancelled
fault	10:17:33.960094 +0200	Docker	CrashReport: report: Failed to set posix_spawn_file_actions for fd -1 at index 0 with errno 9
error	10:17:34.018610 +0200	Docker	SettingsLoader: Cannot load settings: invalidFile
fault	10:17:34.019077 +0200	Docker	Terminator: The operation couldn\U2019t be completed. (Docker.SettingsError error 2.)
default	10:17:34.021477 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
default	10:17:34.021621 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
default	10:17:34.021718 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
default	10:17:34.021809 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
default	10:17:34.021902 +0200	Docker	-[Docker.WizardWindowController loadWindow]: failed to load window nib file '/Applications/Docker.app/Contents/Resources/WizardWindow.nib'.
error	10:17:34.021934 +0200	Docker	Wizard: Wizard.show cannot get window from WizardWindowController
default	10:17:34.063503 +0200	com.docker.osxfs	Unexpected exception: Socket.Closed
default	10:17:34.063562 +0200	com.docker.osxfs	Unexpected exception: Socket.Closed
default	10:17:39.524092 +0200	com.docker.hyperkit	
The system is going down NOW!
default	10:17:39.524564 +0200	com.docker.hyperkit	
Sent SIGTERM to all processes
default	10:17:40.531002 +0200	com.docker.hyperkit	
Sent SIGKILL to all processes
default	10:17:40.532000 +0200	com.docker.hyperkit	
Requesting system poweroff
default	10:17:41.536404 +0200	com.docker.hyperkit	[   83.426530] sd 0:0:0:0: [sda] Synchronizing SCSI cache
default	10:17:41.537560 +0200	com.docker.hyperkit	[   83.427969] sd 0:0:0:0: [sda] Stopping disk
default	10:17:41.539357 +0200	com.docker.hyperkit	[   83.429559] ACPI: Preparing to enter system sleep state S5
default	10:17:41.540054 +0200	com.docker.hyperkit	[   83.430801] reboot: Power down
default	10:17:41.555561 +0200	com.docker.vpnkit	Vmnet.Server.listen: read EOF so closing connection
default	10:17:41.555627 +0200	com.docker.vpnkit	Vmnet.Server.disconnect
default	10:17:41.555664 +0200	com.docker.vpnkit	Vmnet.Server.listen returning Ok()
default	10:17:41.555698 +0200	com.docker.vpnkit	TCP/IP stack disconnected
```

**Comment by pawelkaminski-casumo** [comment]
any comments on that? actually with hint to start docker's tty with screen helped to run docker containers and there is communication. but still I cannot access docker's UI etc!

**Comment by ishandrov** [comment]
The same issue here: The operation couldn’t be completed. (Docker.VmnetdError error 1.)

**Comment by zshimanchik** [comment]
https://github.com/docker/for-mac/issues/3816#issuecomment-519829532
exactly same problem

**Comment by jimklo** [comment]
Similar issue... I cannot even run the uninstaller.

**Comment by montykamath** [comment]
Same problem.  No idea.  Reset settings doesn't fix it either.

**Comment by zshimanchik** [comment]
https://github.com/docker/for-mac/issues/3816#issuecomment-519829532
Solved by complete uninstallation via script + deletion all related to docker folders, everything you can find. And then install again
